src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -877,7 +919,7 @@ private async ValueTask AcquireWriteLockAsync(CancellationToken cancellationToke
                         // at least one other pending writer who can handle the flush.  Worst case, we pay for a flush that ends up being
                         // a nop.  Note: we explicitly do not pass in the cancellationToken; if we're here, it's almost certainly because
                         // cancellation was requested, and it's because of that cancellation that we need to flush.
-                        LogExceptions(FlushAsync());
+                        LogExceptions(FlushAsync(default));
```

Comment:
Can this `default` be either removed (this is already an optional param) or have a more explicit `cancellationToken: default`?

Reply:
Ditto